docs.rs homepage examples use `rand_core` crate option, but do not mention its existence

I'm attempting to follow the [documentation](https://docs.rs/ed25519-dalek/latest/ed25519_dalek/#example) and have the following file:
```rs
use ed25519_dalek::SigningKey;
use rand::rngs::OsRng;

fn main() {
  let mut csprng = OsRng;
  let signing_key: SigningKey = SigningKey::generate(&mut csprng);
}
```
which I'm attempting to build against ed25519_dalek 2.1.1 and rand 0.8.5 but I'm getting the following error:
```
error[E0599]: no function or associated item named `generate` found for struct `SigningKey` in the current scope
 --> src/bin/foo.rs:6:45
  |
6 |   let signing_key: SigningKey = SigningKey::generate(&mut csprng);
  |                                             ^^^^^^^^ function or associated item not found in `SigningKey`

For more information about this error, try `rustc --explain E0599`.
error: could not compile `cherami` (bin "foo") due to previous error
```

--- a/ed25519-dalek/src/lib.rs
+++ b/ed25519-dalek/src/lib.rs
@@ -21,6 +21,7 @@
 #![cfg_attr(feature = "rand_core", doc = "```")]
 #![cfg_attr(not(feature = "rand_core"), doc = "```ignore")]
 //! # fn main() {
+//! // $ cargo add ed25519_dalek --features rand_core
 //! use rand::rngs::OsRng;
 //! use ed25519_dalek::SigningKey;
 //! use ed25519_dalek::Signature;

